packages/sdk-react/src/components/payables/PayableDetails/PayableDetailsForm/helpers.ts

```diff
@@ -225,3 +226,30 @@ function formatTaxToMinorUnits(tax: number): number {
 function formatTaxFromMinorUnits(tax: number): number {
   return tax / 100;
 }
+
+export const isFieldRequired = (
+  fieldName: string,
+  ocrRequiredFields: Record<string, boolean> | undefined,
+  value?: string | null
+) => {
+  const defaultRequiredFields: Record<string, boolean> = {
+    invoiceNumber: true,
+    dueDate: true,
+    tags: true,
+    currency: true,
+  };
+
+  if (value) {
+    return false;
+  }
+
+  const isDefaultRequired = defaultRequiredFields[fieldName] ?? false;
+  const isOcrRequired = ocrRequiredFields?.[fieldName] ?? false;
+
+  return isDefaultRequired || isOcrRequired;
+};
+
+export const useOcrFields = () => {
+  const theme = useTheme();
+  return theme.components?.MonitePayableDetailsInfo?.defaultProps || {};
```

Comment:
@vkuprin Shouldn't it be theme.components?.MonitePayableDetailsInfo?.defaultProps?.ocrFields

Reply:
There is a hook called useThemeProps that's better suited for this use case. You can find an example in InvoiceStatusChip